src/main/java/no/digipost/monitoring/logging/LoggerThresholdMetric.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (C) Posten Norge AS
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package no.digipost.monitoring.logging;
+
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.binder.MeterBinder;
+
+import java.time.Duration;
+
+/**
+ *
+ * `new LoggerThresholdMetric("ROOT, 1, "error", Duration.ofMinutes(5))`
+ * Example output:
+ * # HELP log_events_1min_threshold
+ * # TYPE log_events_1min_threshold gauge
+ * log_events_5min_threshold{level="error",logger="ROOT",} 1.0
+ * <p>
+ * <p>
+ * Example error alert:
+ * - alert: MyLoggingAlert
+ * expr: &gt;
+ * sum by (job,name,level,logger) (increase(logback_logger_events_total[5m]))
+ * &gt;=
+ * max by (job,name,level,logger) (log_events_5min_threshold)
+ */
+class LoggerThresholdMetric implements MeterBinder {
+
+    private final String loggerName;
+    private final double threshold;
+    private final String level;
+    private final Duration duration;
+
+    Gauge thresholdGauge;
+
+    LoggerThresholdMetric(String loggerName, double threshold, String level, Duration duration) {
```

Comment:
Den kanske inte borde ta in en `Duration` om det måste vara i minutter? `int` är ju lite kjipt.. men mer korrekt
`int thresholdSpanInMinutes`. Vill man ha penare typing så evt. klasse `Minutes.of(5)` 😄 

Man kan klart engineera lite till och introducera nån sorts typ `ThresholdSpan` som har en `double` value och möjlighet att skriva metrikk-navnet (`min` eller vad det nu ska vara)  😄, men då baller det på sig lite 😆

Reply:
på linje 60 tar jeg imidlertid `duration.toMinutes()`, så man vil alltid få det i minutter..!
Men klart, blir jo enda klarere med `Minutes.of(5)`. Tenker det kan være en fin endring hvis man gjør klassen public evt. Da kan man i samme runde vurdere om man introduserer feks `ThresholdSpan` e.l. (tenker det er utenfor scope i denne PR) 😁

Reply:
Tenkte på hva hvis noen sender in `Duration.ofSeconds(10)` :)

Reply:
Alltså mest att api-et tillåter dig att göra fel

Reply:
Men litt nitpicking alltså 😄

Reply:
Ahh, good point. Av `Duration.ofSeconds(x)` vil man faktisk runde ned til nærmeste minutt, men ja, enig i poenget. Legger til en `Minutes`-klasse

Reply:
7d8c897